_i18n/en.yml

```
@@ -494,7 +494,7 @@ user-guides:
   purchasing-storing: Securely purchasing and storing Monero
   verify-allos: Verify binaries on Linux, Mac, or Windows command line (advanced)
   verify-windows: Verify binaries on Windows (beginner)
-  mine-on-pool: How to mine on a pool with xmr-stak-cpu
+  mine-on-pool: How to mine on a pool
```

```suggestion
  mine-on-pool: How to mine on a pool with XMRig
```
Since the guide is specific to xmrig, maybe would be better to specify it in the titlle.